lib/puppetserver/ca/action/generate.rb

```diff
@@ -120,13 +121,22 @@ def generate_pki(settings, signing_digest, subject_alt_names = '')
           FileSystem.ensure_dir(settings[:cadir])
           FileSystem.ensure_dir(settings[:certdir])
           FileSystem.ensure_dir(settings[:privatekeydir])
+          FileSystem.ensure_dir(settings[:publickeydir])
+
+          FileSystem.write_file(settings[:cert_inventory],
+                                "0x0001 #{format_time(master_cert.not_before)} " +
```

Comment:
nit: this is a weird bit of domain knowledge that would be good to put into a method

Reply:
okay